Dockerfile

```diff
@@ -23,16 +23,21 @@ LABEL org.label-schema.schema-version="1.0" \
 
 # Install dependencies
 RUN apk add --no-cache \
+      apr \
+      apr-util \
       ca-certificates \
       openssl \
       tini \
  && apk add --no-cache --virtual .builddeps \
+      apache2-utils \
       build-base \
       libffi-dev \
       python3-dev \
+ && cp /usr/bin/htpasswd /tmp \
```

Comment:
Is there any reason, why htpasswd is needed? I'd advocate it to use it on the host itself.

Reply:
the needed tool is not always avail. therefore it makes sense to distribute it with the image. it doesn't conflict with what you propose to advocate and i *guess* it's less than a megabyte space consumed.